sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,204 @@
+  - Fixed crash when array of pointers is passed in kernel[1fc0e4f84]
+  - Fixed issues with use of type from std namespace in kernel type names
+    [dd7fec83]
+  - Fixed debug information missed for work-item builtin translation [9c06d429]
+  - Added warnings emission which had been suppressed for SYCL headers [e6eed1a7]
+  - Fixed optimization disabling option for gen to use -cl-opt-disable
+    [ba4e567fe]
+  - Emulated "funnel shift left" which was not supported in the OpenCL
+    ExtInst set on SPIRV translator side [97d7eec5]
+  - Fixed build issue when TBB and libstdc++ 10.X were used caused by including
+    std C++ headers in integration header file [63369132]
+  - Fixed processing for partial link step with static archives by passing linker
+    specific arguments there [3ab8cc82]
+  - Enabled -reuse-exe support for Windows [43f2d4ba]
+  - Fixed missing dependency file when compiling for -fintelfpga and using a named
+    dependency output file [df5f1ab67]
+  - Set default work group size to {1, 1, 1} to fix out-of-memory crashes on
+    some configurations [4d76de43]
+
+### SYCL Library
+  - Fixed build log preserving for L0 plugin [638b71b1]
+  - Added missing math APIs for devicelib [e438bc814]
+  - Enabled async_work_group_copy for scalar and vector bool types [bb78d2cb]
+  - Aligned image class constructors with the SYCL specification [049ae996]
+  - Removed half type alias causing name conflicts with CUDA headers [c00c1fa3]
+  - Fixed explicit copy operation for host device [f20fd4de]
+  - Made stream flush operation non-blocking [e7492fb2]
+  - Fixed image arguments order when passing to PI routines [70d6f87b]
+  - Fixed circular dependency between the device_impl and the platform_impl
+    causing handler leak [255f304f]
+  - Fixed work-group size selection in reductions [2ae49f57e]
+  - Fixed compilation errors when built with --std=c++20 [ecd0adbb]
+  - Fixed treating all allocations of host pointers as read only causing double
+    free issue [8b5506255]
+  - Fixed a problem in Level Zero plugin with kernels and programs destruction
+    while they can be used [b9bf9f5f]
+  - Fixed wrong exception raised by ALLOWLIST mechanism [d81081f7]
+  - Supported GroupBroadcast with 32-bit id to cover broadcast algorithm with
+    the sub_group class [6e3f2440]
+  - Fixed reporting supported device partitioning in Level Zero [766367be]
+  - Aligned get_info<info::device::version>() with the SYCL spec [4644e639]
+
+### Documentation
+  - Fixed path to FPGA device selector [ca33f7f7]
+  - Renamed LEVEL0 environment variable to LEVEL_ZERO in documents and code
+    comments following source code change [2c3908b4]
+  - Clarified --system-ocl key in GetStartedGuide.md [e31b94e5]
+
+## API/ABI breakages
+  - Implemented SYCL_INTEL_device_specific_kernel_queries [24ae95b3]
+  - Implemented accessor_properties extension [f7d073d1]
+  - Added ONEAPI::filter_selector that accepts one or more filters for device
```

Comment:
I can't see why this change is listed as ABI break.
`Adding a new exported symbol is considered to be non-breaking change.`.